eZ/Publish/Core/Persistence/Cache/ContentLanguageHandler.php

```diff
@@ -119,9 +119,9 @@ function ($languageCode) {
     public function loadListByLanguageCodes(array $languageCodes): iterable
     {
         return $this->getMultipleCacheValues(
-            $languageCodes,
+            array_map([$this, 'escapeForCacheKey'], $languageCodes),
             'ez-language-code-',
-            function (array $languageCodes) {
+            function () use ($languageCodes) {
```

Comment:
Is this change necessary?

Reply:
without this change, we will use the variable after using the escapeForCacheKey function, and we need the original value to search in the database